cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -265,6 +266,20 @@ public IMeasurementSchema getSeriesSchema(PartialPath fullPath) throws MetadataE
     return super.getSeriesSchema(fullPath, getMeasurementMNode(fullPath));
   }
 
+  /**
+   * Transform the PartialPath to VectorPartialPath if it is a sub sensor of one vector. otherwise,
+   * we don't change it.
+   */
+  @Override
+  public PartialPath transformPath(PartialPath partialPath) throws MetadataException {
+    MeasurementMNode node = getMeasurementMNode(partialPath);
```

Comment:
I think `getMeasurementMNode`  should be consist with `getNodeByPath`... 
However, this is a problem carried over from the past..

Reply:
> I think `getMeasurementMNode` should be consist with `getNodeByPath`...
To be consistent, do you mean that method `MManager.getNodeByPath()` should have the function of getting schema both locally and remotely? Or method `CMManager.getMeasurementMNode()`should only have the function of getting schema locally?

Reply:
The diff of`transformPath`  between CMManager and MManager is the `getMeasurementMNode`.
Therefore, I think a better way is to re-write  `getMeasurementMNode` in MManager, which just retrieves the Node locally, rather than to use  `getNodeByPath ` in MManager's `transformPath` and  use `getMeasurementMNode` in CMManger.